### PR TITLE
Devices refactor

### PR DESCRIPTION
**Pull request by kwm4385**
Refactors Devices to use a simpler drag and drop system. Also enables editing of devices.

### CODE REVIEW

File: src/components/shared/InlineEditable/InlineEditable.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default class InlineEditable extends React.Component {
 
   handleSave() {
     const data = {};
-    data[this.props.name.toLowerCase()] = this.state.editorValue;
+    data[this.props.name] = this.state.editorValue;
```
Comment (paradime) [inline src/components/shared/InlineEditable/InlineEditable.js]: should we add a prop check?

File: src/routes/Profile/components/Devices/Device.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      case 'email': return 'mail';
+      case 'sms': return 'comment';
+      default: return 'user';
+    }
+  }
+
+  render() {
+    const device = this.props.device;
+
+    const time = !this.props.isLast && (
```
Comment (paradime) [inline src/routes/Profile/components/Devices/Device.js]: What are you checking here?

File: src/components/shared/InlineEditable/InlineEditable.js (hunk 1 above)
Comment (kwm4385) [inline src/components/shared/InlineEditable/InlineEditable.js]: added PropTypes to InlineEditable

File: src/routes/Profile/components/Devices/Device.js (hunk 2 above)
Comment (kwm4385) [inline src/routes/Profile/components/Devices/Device.js]: we don't want to show an interval if it's the last device in the chain.

**Comment by kwm4385** [comment]
@paradime Let me know if you have anything else on this, I'd like to merge before the meeting/demo today.

**Pull request merged**